### PR TITLE
Added #website and slack link to feedback

### LINKED ISSUE #561: ✨ Enhancement: Modify the response of the feedback form
### Is your feature request related to a problem? Please describe

As of now we are sharing a generic response with the people that send us feedback in the DocsHelp component. However we'd like to improve it.

Screenshot 2024-03-18 at 19 10 33

### Describe the solution you'd like

A better response will be:

Thanks for your feedback! It will be published in the `#website` channel of our [Slack](https://json-schema.org/slack) workspace. Please join us there to continue the discussion!

If it is too long let's improve it!

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by Sahil-Shadwal**
**What kind of change does this PR introduce?**
This modifies the feedback response now it says :
Thanks for the feedback! See it on [#website](https://json-schema.slack.com/archives/C8C4UBXDF) in [Slack ](https://json-schema.org/slack)for further discussion.

**Issue Number:**

-  Closes #561 

**Screenshots/videos:**
Screenshot from 2024-03-22 13-08-53

**If relevant, did you update the documentation?**

**Summary**

It improves feedback response for better user experience.

**Does this PR introduce a breaking change?**
No it doesn't.

### CODE REVIEW

**Comment by Sahil-Shadwal** [comment]
Screenshot from 2024-03-22 12-56-17
updated message

**Comment by Sahil-Shadwal** [comment]
image

the updated  feedback

File: components/DocsHelp.tsx [hunk 1]
```diff
@@ -214,7 +214,38 @@ export function DocsHelp() {
...
+                Thanks for the feedback! Feel free to join the&nbsp;
+                <a
+                  target='_blank'
+                  rel='noreferrer'
+                  className='underline'
+                  href='https://json-schema.slack.com/archives/C8C4UBXDF'
+                >
+                  #website
+                </a>
+                &nbsp;on&nbsp;
```
Comment (benjagm) [inline components/DocsHelp.tsx:227]:
```suggestion
                &nbsp;channel&nbsp;on&nbsp;
```

File: components/DocsHelp.tsx [hunk 2]
```diff
@@ -214,7 +214,38 @@ export function DocsHelp() {
...
+                <a
+                  target='_blank'
+                  rel='noreferrer'
+                  className='underline'
+                  href='https://json-schema.org/slack'
+                >
+                  <svg
```
Comment (benjagm) [inline components/DocsHelp.tsx:234]: I think we don't need this icon here, with the text link I think is enough.

**Review by benjagm — CHANGES_REQUESTED** [review]
Great job! Just leave 2 comments!

File: components/DocsHelp.tsx (hunk 1 above)
Comment (Sahil-Shadwal) [reply components/DocsHelp.tsx]: done

File: components/DocsHelp.tsx (hunk 2 above)
Comment (Sahil-Shadwal) [reply components/DocsHelp.tsx]: done

**Review by DhairyaMajmudar — APPROVED** [review]
Amazing LGTM 💯

**Review by benjagm — APPROVED** [review]
Great job! Thanks for this!!

**Pull request merged**